src/main/java/com/cloudbees/jenkins/plugins/bitbucket/client/BitbucketCloudApiClient.java

```diff
@@ -377,13 +377,25 @@ public String getDefaultBranch() throws IOException, InterruptedException {
         return cachedDefaultBranch;
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @NonNull
+    @Override
+    public List<BitbucketCloudBranch> getTags() throws IOException, InterruptedException {
+        return getServerBranches("/refs/tags");
+    }
     /**
      * {@inheritDoc}
      */
     @NonNull
     @Override
     public List<BitbucketCloudBranch> getBranches() throws IOException, InterruptedException {
-        String url = UriTemplate.fromTemplate(REPO_URL_TEMPLATE + "/refs/branches")
+        return getServerBranches("/refs/branches");
+    }
+
+    public List<BitbucketCloudBranch> getServerBranches(String nodePath) throws IOException, InterruptedException {
```

Comment:
`getServerBranches`? This is confusing, could you use a more descriptive name? Maybe `getBranchesByRef(String)`?